:warning: (kustomize/v1) deprecated v1 version in favor of v2 (#3235)
kubernetes-sigs/kubebuilder: docs/book/src/plugins/to-be-extended.md

another languages (i.e. [Operator-SDK][sdk] does to allow users work with Ansible/Helm) or to add
helpers on top, such as [Operator-SDK][sdk] does to add their features to integrate the projects with [OLM][olm].

| Plugin                                                                             | Key                         | Description                                                                                                                                                                                                                                  |
| ---------------------------------------------------------------------------------- |-----------------------------| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | [kustomize.common.kubebuilder.io/v1](kustomize-v1.md)                              | `kustomize/v1 (Deprecated)` | Responsible for scaffold all manifests to configure the projects with [kustomize(v3)][kustomize]. (create and update the `config/` directory). This plugin is used in the composition to create the plugin (`go/v3`).                    |

Reviewer: Nit ```suggestion
| [kustomize.common.kubebuilder.io/v1](kustomize-v1.md)                              | `kustomize/v1 (Deprecated)` | Responsible for scaffolding all manifests to configure projects with [kustomize(v3)][kustomize]. (create and update the `config/` directory). This plugin is used in the composition to create the plugin (`go/v3`).                    |
```

> | [kustomize.common.kubebuilder.io/v1](kustomize-v1.md)                              | `kustomize/v1 (Deprecated)` | Responsible for scaffold all manifests to configure the projects with [kustomize(v3)][kustomize]. (create and update the `config/` directory). This plugin is used in the composition to create the plugin (`go/v3`).                    |

Author: ```suggestion
| [kustomize.common.kubebuilder.io/v1](https://github.com/kubernetes-sigs/kubebuilder/pull/3235/kustomize-v1.md) | kustomize/v1 (Deprecated) | Responsible for scaffolding all manifests to configure projects with [kustomize(v3)][kustomize]. (create and update the `config/` directory). This plugin is used in the composition to create the plugin (`go/v3`). |
```

> | [kustomize.common.kubebuilder.io/v2-alpha](kustomize-v2-alpha.md)                  | `kustomize/v2-alpha`        | It has the same purpose of `kustomize/v1`. However, it works with [kustomize][kustomize] version `v4` and addresses the required changes for future kustomize configurations. It will probably be used with the future `go/v4-alpha` plugin. |

Reviewer: ```suggestion
| [kustomize.common.kubebuilder.io/v2-alpha](kustomize-v2-alpha.md)                  | `kustomize/v2-alpha`        | It has the same purpose as of `kustomize/v1`. However, it works with [kustomize][kustomize] version `v4` and addresses the required changes for future kustomize configurations. It will probably be used with the future `go/v4-alpha` plugin. |
```

> | `base.go.kubebuilder.io/v3`                                                        | `base/v3`                   | Responsible for scaffold all files which specific requires Golang. This plugin is used in the composition to create the plugin (`go/v3`)                                                                                                     |

Reviewer: ```suggestion
| `base.go.kubebuilder.io/v3`                                                        | `base/v3`                   | Responsible for scaffolding all files that specifically require Golang. This plugin is used in composition to create the plugin (`go/v3`)                                                                                                     |
```
